src/components/navBar/navBar.spec.js

```diff
@@ -156,6 +156,17 @@ describe('mdNavBar', function() {
 
       expect($scope.selectedTabRoute).toBe('tab2');
     });
+
+    it('does not update tabs if tab controller is undefined', function() {
+      $scope.selectedTabRoute = 'tab1';
+
+      spyOn(Object.getPrototypeOf(ctrl), '_updateInkBarStyles');
```

Comment:
Pretty sure the Object.getPrototypeOf indirection is unnecessary. Can you remove the 3 instances in this test?